README.adoc

```diff
@@ -2436,7 +2430,7 @@ endif::[]
 
 === Tearing down the environment 
 
-When you're finished trying out the microservice, press **CTRL+C** in the command line session where you ran the `kubectl port-forward` command to stop the port forwarding. You can delete all Kubernetes resources by running the `kubectl delete` commands:
+When you're finished trying out the microservice, press `CTRL+C` in the command line session where you ran the `kubectl port-forward` command to stop the port forwarding. You can delete all Kubernetes resources by running the `kubectl delete` commands:
```

Comment:
just to confirm is it `CTRL+C` or is it `CTRL+c` or `ctrl+c` ?

Reply:
actually is c. Because used C across all guides, we will update together if need.